python/tests/test_0_complex_or_slow.py

```diff
@@ -668,3 +669,14 @@ def test_verified_group_vs_delete_server_after(acfactory, tmp_path, lp):
     assert msg_in.chat == chat2_offl
     assert msg_in.get_sender_contact().addr == ac2.get_config("addr")
     assert ac2_offl_ac1_contact.is_verified()
+
+
+def test_deleted_msgs_dont_reappear(acfactory):
+    (ac1,) = acfactory.get_online_accounts(1)
```

Comment:
`bcc_self` doesn't need to be set, we anyway send the message to self. But it's an interesting question: why we do so? Maybe if we send a message to self and `bcc_self` is disabled, just store it locally?

Reply:
I agree. Anyway, why we smtp-send self-messages at all if `bcc_self` is unset? This looks like an exception from what we have for chats in general (1:1, group, etc.)